.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
```

Comment:
If this configuration is meant for a specific environment or setup, it would be helpful to add a comment explaining the purpose of this configuration, especially for new developers.